gcm_toolkit/exorad/read_in.py

```diff
@@ -119,6 +187,15 @@ def m_read_from_mitgcm(
     if loaded_dsi is not None:
         dsi = xr.merge([dsi, loaded_dsi])
 
+    # change the names of the cloud variables
+    if 'ClAb' in dsi.keys():
+        dsi = dsi.rename({'Cl01': 'ClVf_TiO2[s]', 'Cl02': 'ClVf_Mg2SiO4[s]', 'Cl03': 'ClVf_SiO[s]',
+                          'Cl04': 'ClVf_SiO2[s]', 'Cl05': 'ClVf_Fe[s]', 'Cl06': 'ClVf_Al2O3[s]',
+                          'Cl07': 'ClVf_CaTiO3[s]', 'Cl08': 'ClVf_FeO[s]', 'Cl09': 'ClVf_FeS[s]',
+                          'Cl10': 'ClVf_Fe2O3[s]', 'Cl11': 'ClVf_MgO[s]', 'Cl12': 'ClVf_MgSiO3[s]',
+                          'Cl13': 'ClVf_CaSiO3[s]', 'Cl14': 'ClVf_Fe2SiO4[s]', 'Cl15': 'ClVf_C[s]',
+                          'Cl16': 'ClVf_KCl[s]'})
+
```

Comment:
This makes no sense to me, what do you aim to achieve with it? Is this easier than just setting these keys above?

Reply:
Within the MITgcm, the cloud species are hard coded with numbers and have to stay that way.
Within gcmt, the idea is to have a more flexible approach to allow for multiple different species.

I did not find a way to read in, for example, Cl01 as ClVf_TiO2[s], but if you know how, please let me know!

Reply:
ahhhhh..... now i get it. Ok thanks!